src/java/org/apache/cassandra/service/paxos/cleanup/PaxosCleanupHistory.java

```diff
@@ -47,21 +51,24 @@ public void serialize(PaxosCleanupHistory message, DataOutputPlus out, int versi
             message.tableId.serialize(out);
             message.highBound.serialize(out);
             PaxosRepairHistory.serializer.serialize(message.history, out, version);
+            Epoch.messageSerializer.serialize(message.epoch, out, version);
```

Comment:
this will need a version check. There will already be history entries serialized without an epoch in the wild

Reply:
Added missing version check